netjsonconfig/backends/openwrt/converters/__init__.py

```diff
@@ -11,15 +12,16 @@
 from .wireless import Wireless
 
 __all__ = [
-    'Default',
-    'Interfaces',
-    'General',
-    'Led',
-    'Ntp',
-    'OpenVpn',
-    'Radios',
-    'Routes',
-    'Rules',
-    'Switch',
-    'Wireless',
+    "Default",
```

Comment:
Disagree - black, the linter used by the project, is advocating the use of double quotes, so anywhere I am touching code I'll go with what the linter prefers.

Reply:
@jonathanunderwood black was added recently and we didn't change the quoting style. The formatting tool we use is `openwisp-qa-format` which has the rule to skip changing the quoting style. We should avoid changing lines of code which do not need to be changed (because unnecessary changes make it harder to track bugs with git blame) and we should follow consistency with the existing codebase as much as possible.

Reply:
You seriously need a coding standards document to clarify this. Honestly, with all the work that's going into  this, this is really immaterial, and this nitpicking is seriously demotivating.